From 58bf006646a063837c46b695f7e7ebb69bee7238 Mon Sep 17 00:00:00 2001
From: Mark Hatle <mark.hatle@windriver.com>
Date: Tue, 15 Aug 2017 16:41:57 -0500
Subject: [PATCH] perl: disable auto requires

When generating automatic requirements, it's possible for perl scripts to
declare 'optional' dependencies.  These seem to often be incorrect and will
cause installation failures in OE.  Instead of fixing the perl scripts, it
was decided it is better to simply disable the automatic dependency
generation.  This matches the behavior from the previous RPM5 implementation.

Upstream-Status: Inappropriate [OE specific configuration]

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>
---
 fileattrs/perl.attr    | 2 +-
 fileattrs/perllib.attr | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fileattrs/perl.attr b/fileattrs/perl.attr
index 0daef58d5..81ddf5305 100644
--- a/fileattrs/perl.attr
+++ b/fileattrs/perl.attr
@@ -1,3 +1,3 @@
-%__perl_requires	%{_rpmconfigdir}/perl.req
+#__perl_requires	%{_rpmconfigdir}/perl.req
 %__perl_magic		^.*[Pp]erl .*$
 %__perl_flags		exeonly
diff --git a/fileattrs/perllib.attr b/fileattrs/perllib.attr
index fcad48099..495a28927 100644
--- a/fileattrs/perllib.attr
+++ b/fileattrs/perllib.attr
@@ -1,5 +1,5 @@
 %__perllib_provides	%{_rpmconfigdir}/perl.prov
-%__perllib_requires	%{_rpmconfigdir}/perl.req
+#__perllib_requires	%{_rpmconfigdir}/perl.req
 %__perllib_magic	^Perl[[:digit:]] module source.*
 %__perllib_path		\\.pm$
 %__perllib_flags	magic_and_path
